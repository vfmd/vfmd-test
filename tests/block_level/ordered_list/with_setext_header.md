 1. List item
    List item

    Part of the list
    ======

 1. List item
    List item

    Part of the list
======

 1. List item
    List item
    Part of the list
======

 1. List item
    List item
Part of the list
======

 1. List item
    List item

Not part of the list
======

 1. List item
    List item

    Part of the list
    ------

 1. List item
    List item

    Part of the list
------

 1. List item
    List item
    Part of the list
------

 1. List item
    List item
Part of the list
------

 1. List item
    List item

Not part of the list
------

