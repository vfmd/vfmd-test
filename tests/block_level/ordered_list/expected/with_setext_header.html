
<ol>
  <li>
    <p>List item
List item</p>
    <h1>Part of the list</h1>
  </li>
  <li>
    <p>List item
List item</p>
    <h1>Part of the list</h1>
  </li>
  <li><p>List item
List item
Part of the list
======</p></li>
  <li><p>List item
List item
Part of the list
======</p></li>
  <li><p>List item
List item</p></li>
</ol>
<h1>Not part of the list</h1>

<ol>
  <li>
    <p>List item
List item</p>
    <h2>Part of the list</h2>
  </li>
  <li>
    <p>List item
List item</p>
    <p>Part of the list</p>
  </li>
</ol>
<hr />

<ol>
  <li>List item
List item
Part of the list</li>
</ol>
<hr />

<ol>
  <li>List item
List item
Part of the list</li>
</ol>
<hr />

<ol>
  <li>List item
List item</li>
</ol>
<h2>Not part of the list</h2>
